packages/stacked_themes/lib/src/theme_manager.dart

```diff
@@ -50,7 +50,7 @@ class ThemeManager {
   /// Returns the current selected theme mode
   ThemeMode? get selectedThemeMode => _selectedThemeMode;
 
-  ThemeModel get initalTheme => _initialTheme!;

```

Comment:
Any specific reason for this change. It just looks like a change in the variable name. For projects using linting, this will cause issues since the spelling is wrong.